### PR TITLE
closes #8 , crawler to clone site

### LINKED ISSUE #8: Create API k8-Crawler service (Website Crawler).
Create a service that crawls a website and creates an offline version of it.

### PR DESCRIPTION
**Pull request by belwase**
- closes #8 
- service to make local copy of a website
- flask endpoint to run the service
- Demo video - https://drive.google.com/file/d/1Ra9d7VD83KZj5PlOLE7cr6TgErDnDM6_/view?usp=sharing

### CODE REVIEW

File: backend/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      MINIO_URL: http://172.22.0.2:9000
+      MINIO_ACCESS_KEY: minio1
+      MINIO_SECRET_KEY: minio1@123
+    env_file:
+      - .env
+    ports:
+      - 5000:5000
+
+  minio:
+    image: minio/minio:RELEASE.2020-09-08T23-05-18Z
```
Comment (manish2kcoder) [inline backend/docker-compose.yaml:18]: Hi @belwase just curious, what is our intent beind using minio here? the static site will be hosted on flask? will minio hold the files that will be scrapped from gov.uk? in that case how we plan to rewrite the site urls to fetch those file from minio?

Comment (belwase) [reply backend/docker-compose.yaml:18]: @manish2kcoder  After crawling all files from website, we'll extract specific files to storage service ( minio) which will be used by analysis part.

Comment (belwase) [reply backend/docker-compose.yaml:18]: FYI we won't host static site, we'll have our own app to be hosted . the site that we crawl will only be used for analysis

Comment (manish2kcoder) [reply backend/docker-compose.yaml:18]: ok, actually a comment in stack says:

Nadege Hode (GW-Test Team Lead)  8:06 PM
@Ashish ( Upwork - Senior Python Engineer ), @Jonathan Canales (Upwork - Tech PM), we need a saved copy of the whole gov.uk website and be able to access that for performance testing; not just to get the files

So what is expected from the page we host is still not clear to me, let us talk in AMA never mind.

**Review by DinisCruz — APPROVED** [review]

**Pull request merged**